code/game/verbs/suicide.dm

```diff
@@ -71,15 +65,10 @@
 					adjustOxyLoss(max(175 - getToxLoss() - getFireLoss() - getBruteLoss() - getOxyLoss(), 0))
 
 				updatehealth()
-				return
-
 
-		visible_message(pick(span_danger("[src] is attempting to bite [p_their()] tongue off! It looks like [p_theyre()] trying to commit suicide."), \
-							span_danger("[src] is jamming [p_their()] thumbs into [p_their()] eye sockets! It looks like [p_theyre()] trying to commit suicide."), \
-							span_danger("[src] is twisting [p_their()] own neck! It looks like [p_theyre()] trying to commit suicide."), \
-							span_danger("[src] is holding [p_their()] breath! It looks like [p_theyre()] trying to commit suicide.")))
-		adjustOxyLoss(max(175 - getToxLoss() - getFireLoss() - getBruteLoss() - getOxyLoss(), 0))
-		updatehealth()
+				suiciding = FALSE	//Should have been set to FALSE because if you were revived or healed, you could not suicide again
```

Comment:
This var is used to determine whether the mob can be revived or not. FALSEing it here makes suicides defibable.

Reply:
Is being revivable not intended? I saw the little yellow lightning bolt indicating you could be revived when I was testing it out.

Reply:
I mean, the defib specifically checks for it. That's pretty intended. The medhud icon update just doesn't account for it.

Reply:
Alright, that begs the question though. Should suicides be revivable? I don't see an issue with it. Also, the FALSE was needed as suicide did not instantly kill you so if you healed up before you died you could not suicide again. If people want suicides to be unrevivable instant kills I will need to rework them.

Reply:
That sounds like it's just an issue with suiciding not doing enough damage. You could maybe set stat directly, but that might have awkward consequences.
If suicides are revivable, it opens them up as a strat since once you're dead xenos can't harm or move you.

Reply:
Felt thematic suicides not killing you instantly, since some range from slitting wrists to strangulation. This could change with a rework by just dealing enough damage to kill the person instantly though.

Tactical suicide is not ideal since revives do heart damage and it's not with just any item you can do it with. As for your body not being movable due to it, that's corpse dragging being a problem not tactical suicides.